extension/chrome/elements/compose-modules/compose-input-module.ts

```diff
@@ -36,11 +36,11 @@ export class ComposeInputModule extends ViewModule<ComposeView> {
   }
 
   public setHandlers = () => {
+    this.toggleRichTextFormatting();
```

Comment:
`toggleRichTextFormatting` sounds like we change rich text property here, but we just configure textarea according to the existing property value. 
`configureRichTextFormatting` is more correct naming, as for me.

Reply:
Ah, I see. I agree with your suggestion and have changed the name as recommended. Thanks!